WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt

```diff
@@ -75,7 +77,7 @@ class HomePagePickerFragment : Fragment() {
         viewModel = ViewModelProviders.of(requireActivity(), viewModelFactory)
                 .get(HomePagePickerViewModel::class.java)
 
-        viewModel.uiState.observe(viewLifecycleOwner, Observer { uiState ->
+        viewModel.uiState.observe(viewLifecycleOwner, { uiState ->
```

Comment:
For some reason, AS wasn't happy until I re-added the explicit type here:

```diff
diff --git a/WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt b/WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt
index a67a1d8d4f..34404dba03 100644
--- a/WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt
+++ b/WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt
@@ -6,6 +6,7 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.ViewModelProviders
 import androidx.recyclerview.widget.GridLayoutManager
@@ -77,7 +78,7 @@ class HomePagePickerFragment : Fragment() {
         viewModel = ViewModelProviders.of(requireActivity(), viewModelFactory)
                 .get(HomePagePickerViewModel::class.java)
 
-        viewModel.uiState.observe(viewLifecycleOwner, { uiState ->
+        viewModel.uiState.observe(viewLifecycleOwner, Observer { uiState ->
             setTitleVisibility(uiState.isHeaderVisible)
             description?.visibility = if (uiState.isDescriptionVisible) View.VISIBLE else View.INVISIBLE
             loadingIndicator.setVisible(uiState.loadingIndicatorVisible)
@@ -96,7 +97,7 @@ class HomePagePickerFragment : Fragment() {
             }
         })
 
-        viewModel.onPreviewActionPressed.observe(viewLifecycleOwner) { action ->
+        viewModel.onPreviewActionPressed.observe(viewLifecycleOwner, Observer { action ->
             activity?.supportFragmentManager?.let { fm ->
                 when (action) {
                     is Show -> {
@@ -108,7 +109,7 @@ class HomePagePickerFragment : Fragment() {
                     }
                 }
             }
-        }
+        })
 
         viewModel.start()
     }
```

Any ideas what could be different about our config?

Reply:
I updated to AS4.1.1 a couple of days ago and started seeing those as an optimisation warning.

![Screenshot 2020-11-17 at 7 52 35 AM](https://user-images.githubusercontent.com/304044/99351759-00ecc380-28aa-11eb-85ff-6176cd4d41a0.png)

I didn't thought that this would cause issues with the previous AS versions. Sorry for the inconvenience.
I would probably put those back.

Reply:
Ah, that explains it. In the meantime, I'll update my AS then (I'm still on 4.0.2). :smile:

Reply:
Fixed with 1c8cd3e3a71b2707b0e323a60966164db4d8188f